.github/workflows/automatic_pull_request_review.yml

```
@@ -0,0 +1,82 @@
+name: Automatic pull request review
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  review_code:
+    name: Run Danger 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Load Gradle cached
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      # Run Detekt and generate Detekt reports
+      - name: Run Detekt on CoroutineTemplate
+        working-directory: ./CoroutineTemplate
+        run: ./gradlew detekt
+
+      - name: Run Detekt on RxJavaTemplate
+        working-directory: ./RxJavaTemplate
+        run: ./gradlew detekt
+
+      # Run Android Lint and generate Lint reports
+      - name: Run Android Lint on RxJavaTemplate
+        working-directory: ./RxJavaTemplate      
+        run: ./gradlew lint
+
+      - name: Run Android Lint on CoroutineTemplate
+        working-directory: ./CoroutineTemplate      
+        run: ./gradlew lint
+
+      # TODO: Update to support test coverage report from Jacoco for Coroutine Template
+      # Run Unit Test & Jacoco for code coverage for RxJavaTemplate
+      - name: Run Unit Tests & Jacoco on RxJavaTemplate
+        working-directory: ./RxJavaTemplate
+        run: ./gradlew jacocoTestReport
+
+      # Setup Ruby gems and Bundle for executing danger-ruby
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+
+      - name: Cache Gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Bundle Install and Version Check
```

```suggestion
      - name: Bundle install and version check
```

I think this should be Capitalize more than Titleize. Same as the workflow name and [`Pull request review by Danger`](https://frontside.com/blog/2020-05-26-github-actions-pull_request/#:~:text=When%20a%20Github%20Actions%20workflow,opened%20%2C%20synchronize%20%2C%20and%20reopened%20.)